BappManifest.bmf

```diff
@@ -2,11 +2,11 @@ Uuid: 6bf7574b632847faaaa4eb5e42f1757c
 ExtensionType: 1
 Name: OpenAPI Parser
 RepoName: openapi-parser
-ScreenVersion: 4.0
+ScreenVersion: 4.1
 SerialVersion: 24
 MinPlatformVersion: 6
 ProOnly: False
-Author: Alexandre Teyar, Aegis Cyber www.aegiscyber.co.uk
+Author: Alexandre Teyar, Aegis Cyber <www.aegiscyber.co.uk>
```

Comment:
Unfortunately, we will have to remove these angled brackets before we publish the extension - we will do this on our side, you will not need to make any changes.